emily/handler/src/api/handlers/new_block.rs

```diff
@@ -336,6 +336,7 @@ fn handle_withdrawal_create(
         request_id: event.request_id,
         stacks_block_hash: event.block_id.to_hex(),
         stacks_block_height,
+        txid: event.txid.to_string(),
```

Comment:
nit: We should prefer something like `.to_hex()` since it is clear what is being stored by looking at the code. It doesn't change what actually happens under the hood, since the `std::fmt::Display` implementation uses `.to_hex()`, it's just clearer.